### ISSUE TITLE
Days highlight

### ISSUE THREAD

**Issue by kirillusgadus** [logisticinfotech/ionic4-datepicker#23]
Please, add days highlight like this
![Снимок экрана 2019-06-13 в 10 47 01](https://user-images.githubusercontent.com/19182192/59413661-aa33ae00-8dc8-11e9-90ad-cd88e451f8fb.png)

**Comment by chintan13** [comment]
ok, 
would you like to elaborate/suggest like
what should be option name?
How would you like to pass it to component?

On what are the cases its useful.. 

So then we will plan out to implement this feature accordingly.

**Comment by kirillusgadus** [comment]
in my case need when the user opens calendar and seen which days there are high or low load, for example, in colors (green, yellow, red). Param like this:
```
highlightedDays: [
    { day: moment(), color: '#dedede' },
]
```

**Comment by chintan13** [comment]
Implemented this in version 1.4.0
